p2p/src/p2p_config.rs

```diff
@@ -110,6 +111,7 @@ impl Default for P2pTimeouts {
                 Some(Duration::from_secs(5)),
             ),
             select: from_env_or("SELECT_TIMEOUT", Some(Duration::from_secs(5))),
+            pnet: Some(Duration::from_secs(5)),
```

Comment:
Please make it configurable via env. BTW, isn't 5 secs too much for PNET?

Reply:
Sure, I'll make it configurable via env var like you suggest.
Five secs is probably too much but I wanted to be on the safe side for some of the tests, maybe we can reduce the default value now and increase it via env var for the tests.